src/main/java/cs2103/v15_1j/jimjim/controller/JJController.java

```diff
@@ -45,6 +45,7 @@ public String getFilePath() {
 	
 	@Override
 	public void setFilePath(String filePath) {
+		this.states.config.savePath = filePath;
```

Comment:
Have to save this config too. And if they fail undo everything and return an error message. Maybe it's better to just create a SaveLocationCommand and execute it here.